pom.xml

```diff
@@ -329,8 +329,8 @@
         <spring.osgi.version>1.2.1</spring.osgi.version>
         <spring31.version>3.1.4.RELEASE</spring31.version>
         <spring43.version>4.3.30.RELEASE_1</spring43.version>
-        <spring52.version>5.2.13.RELEASE_1</spring52.version>
-        <spring53.version>5.3.14_1</spring53.version>
+        <spring52.version>5.2.20.RELEASE_1</spring52.version>
+        <spring53.version>5.3.18_1</spring53.version>
```

Comment:
5.3.18_1 bundle is not available yet from Servicemix.

Reply:
Hopefully, they'll release it soon